docs/getting-started/debugging.md

```diff
@@ -21,8 +21,7 @@ To use the debug gem:
   debugger console.
 4. Use debug commands, which can be found [here](https://github.com/ruby/debug#debug-command-on-the-debug-console).
 
-#### Example
-
+<h3 data-toc-skip class="h4">Example</h3>
```

Comment:
```suggestion
### Example
{:toc-skip}
```
See comment on community/index.md regarding use of Kramdown attribute list definitions to avoid inline HTML.